frontend/packages/helm-plugin/integration-tests/features/helm/install-helm-chart.feature

```diff
@@ -82,3 +82,51 @@ Feature: Install the Helm Release
              When user selects the Chart Version "0.2.0 / App Version 1.16.0 (Provided by Red Hat Helm Charts)"
              When user selects "Proceed" button from Change Chart version confirmation dialog
              Then user will see that the README is also updated with new chart version "0.2.0 / App Version 1.16.0 (Provided by Red Hat Helm Charts)"
+
+
+        @regression
+        Scenario: Certification filter in Helm Catalog Page
+            Given user is at Add page
+              And user has added multiple helm charts repositories with providerType annotations in index.yaml
```

Comment:
Can you please add some test data, otherwise it is very difficult to do the automation

Reply:
Added below the scenario as a comment